### PR TITLE
Create note listing page UI

### LINKED ISSUE #1: Create note listing page UI

### PR DESCRIPTION
**Pull request by vysakh-poulose**
Fixes #1 

- Updated prettier
- Added sidebar, menubar and Header
- Displayed list of notes
- Changed neetoui to neetoui/v2 in alias

### CODE REVIEW

File: app/javascript/src/components/Dashboard/Notes/MenuBar.jsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+import { MenuBar as NeetoMenuBar } from "neetoui/layouts";
+
+const MenuBar = ({ showMenu }) => {
+  return (
+    <NeetoMenuBar showMenu={showMenu} title="Contacts">
+      <NeetoMenuBar.Block label="All" count={200} active />
+      <NeetoMenuBar.Block label="Users" count={80} />
+      <NeetoMenuBar.Block label="Leads" count={60} />
+      <NeetoMenuBar.Block label="Visitors" count={60} />
+
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/MenuBar.jsx:14]: Please refrain from adding blank lines like these within the component JSX.

**Review by yedhink — COMMENTED** [review]
@vysakh-poulose 

Few comments.

File: app/javascript/src/components/Dashboard/Account/Profile/index.jsx [hunk 2]
```diff
@@ -1,9 +1,9 @@
 import React, { useState } from "react";
```
[the remark this answers is not in the document]
Comment (vysakh-poulose) [reply app/javascript/src/components/Dashboard/Account/Profile/index.jsx:1]: Fixed in 3254ea6

File: app/javascript/src/components/Dashboard/Notes/List/Card.jsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
+import React from "react";
+
+import { Clock, MenuVertical } from "neetoicons";
+import { Typography, Tag, Avatar, Tooltip, Dropdown } from "neetoui";
+
+const Card = ({ title, description, tag }) => {
+  return (
```
[the remark this answers is not in the document]
Comment (vysakh-poulose) [reply app/javascript/src/components/Dashboard/Notes/List/Card.jsx:7]: Fixed in b22fe06

File: app/javascript/src/components/Dashboard/Notes/List/constants.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+const NOTES_RANGE = Array.from({ length: 4 }, (_, index) => index + 1);
+
+export const NOTES = NOTES_RANGE.map(note => {
+  return {
```
[the remark this answers is not in the document]
Comment (vysakh-poulose) [reply app/javascript/src/components/Dashboard/Notes/List/constants.js:4]: Fixed in b22fe06

**Pull request merged**